src/transformers/models/wav2vec2_with_lm/processing_wav2vec2_with_lm.py

```diff
@@ -450,6 +450,7 @@ def decode(
         unk_score_offset: Optional[float] = None,
         lm_score_boundary: Optional[bool] = None,
         output_word_offsets: bool = False,
+        n_best: Optional[int] = 1,
```

Comment:
As `n_best` cannot be `None` 

```suggestion
        n_best: int = 1,
```